dvc/config.py

```diff
@@ -135,6 +136,7 @@ class RelPath(str):
     "core": {
         "remote": Lower,
         "checksum_jobs": All(Coerce(int), Range(1)),
+        "jobs": All(Coerce(int), Range(1)),
```

Comment:
Let's not add this one, as it might be harmful for some remotes.
```suggestion
```

Reply:
Could you please explain the usage difference between `"core"` and `REMOTE_COMMON`. I found that they are not always overlapping (used together) and to me, it looked like `"jobs"` field has to be in `"core"` as well for the cases when it used not extended with the `REMOTE_COMMON`. I definitely hadn't investigated all the architecture of the configs. So, I'm wondering, why is it ok to have the `"jobs"` field only in the `REMOTE_COMMON`?

Reply:
Not sure why it was marked as resolved. Unresolving...

@I159 `core` is a common section for everything. Like `core` section in git config. `REMOTE_COMMON` is a common set of options used in `remote *` sections, like in git config. Putting jobs (which only have sense for remotes) into the core section doesn't make much sense, as it is not generic enough to be applied to every remote type and might break some. There are some usecases where it could be useful, but I would rather not add it until someone asks for it.